tests/pdarray_creation_test.py

```diff
@@ -124,6 +124,11 @@ def test_randint(self):
         self.assertEqual(ak.bool, test_array.dtype)
         
         test_ndarray = test_array.to_ndarray()
+
+        # test resolution of modulus overflow - issue #1174
+        test_array = ak.randint(-(2**63), 2**63-1, 10)
+        to_validate = np.full(10, -(2**63))
```

Comment:
on a second look this doesn't seem correct, looking at the issue.
you should get values across the entire range [-(2**63), 2**63-1] and not just the value of (-(2**63))

Reply:
I believe the assertion here is testing that the output is not `-(2**63)` repeated `size` times

```python
self.assertFalse((test_array.to_ndarray() == to_validate).all())
```

Reply:
Just to confirm, what @pierce314159 stated was my intention with the test.